### ISSUE TITLE
new way of using cypress?

### ISSUE THREAD

**Issue by minju25kim** [fac24/week6-bereket-hussain-joe-sonia#35]
E2E is end-to-end test(googled it : ) ), and is it new way to use cypress? Just wonder

**Comment by joe-dev-public** [comment]
Yeah, Cypress recently moved from version 9 (which we were using a few weeks ago) to v10. Looks like they changed quite a lot.

So now when you set it up, it asks you to choose E2E vs another kind of testing, and it sets up different default directory/file structure for you.

We just have a template test to make sure it works :)
